laravel/app/Inventorie.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Inventorie extends Model
+{
+	public function scopeStockIndex($query, $item_id, $delivery_user_id, $status)
+    {
+        $query->join('items', 'inventories.item_id', '=', 'items.item_id');
+
+      	if (isset($item_id)) {
+		    $query->where('inventories.item_id', $item_id);
+		}
+
+		if (isset($delivery_user_id)) {
+		    $query->where('delivery_user_id', $delivery_user_id);
+		}
+
+		if (isset($status)) {
+			$query->where('status', $status);
+		}
+
+		return $query;
+    }
+
+    public function item()
+    {
+    	return $this->belongsTo('App\Item', 'item_id', 'item_id');
+    }
+
+    public function clientCompany() 
+    {
+    	return $this->belongsTo('App\ClientCompany', 'delivery_user_id','id');
```

Comment:
細かいですが、 `,` の後は半角スペースを入れる癖をつけましょう
```suggestion
    	return $this->belongsTo('App\ClientCompany', 'delivery_user_id', 'id');
```